dxr/plugins/clang/indexers.py

```diff
@@ -178,7 +178,7 @@ def get_scopes_to_members():
                 # First, link to the type definition itself:
                 links = [(type['kind'],
                           type['name'],
-                          '#%s' % type['span'].start.row)]
+                          '#%s' % type['span'].start.row, False)]
```

Comment:
It's not your fault; it's mine for getting behind in reviews. But regions and refs now use actual classes for their return values, so let's take the opportunity to do so for links as well. Then, the next time we add something, we won't need to touch every line that deals with the data structure. :-)